packages/react-components/src/molecules/AnalyticsControls.tsx

```diff
@@ -123,9 +132,12 @@ const generateLink = (
   tagsQueryString?: string,
   timeRange?: string,
   documentCategory?: string,
+  outputType?: string,
 ) =>
   `${href}?range=${timeRange}${
     documentCategory ? `&documentCategory=${documentCategory}` : ''
+  }${
+    outputType ? `&outputType=${outputType}` : ''
   }&currentPage=${currentPage}${tagsQueryString}`;
 
 const updateSearchParams = (): URLSearchParams => {
```

Comment:
here the outputType searchParam should be deleted otherwise switching between output types cause multiple params in the url
<img width="991" alt="Screenshot 2024-06-27 at 12 11 03" src="https://github.com/yldio/asap-hub/assets/25244556/c3a4eead-2060-4a3f-ae8e-4cccf672575b">